buildpipeline/Core-Setup-RHEL7-x64.json

```diff
@@ -22,6 +22,29 @@
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
+      "displayName": "Initialize Docker",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "6c731c3c-3c68-459a-a5c9-bde6e6595b5b",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        // init-docker.sh was manually copied from dotnet/buildtools, as opposed to restoring buildtools package.
```

Comment:
I think Newtonsoft.Json handles comments ok, but they aren't part of the JSON spec. I don't think we should consider it valid in the future.